integration_tests/follow_me.cpp

```diff
@@ -37,22 +37,22 @@ TEST_F(SitlTest, FollowMe)
     ASSERT_EQ(Action::Result::SUCCESS, action_ret);
 
     device.telemetry().flight_mode_async(
-        std::bind(&print_flight_mode, std::placeholders::_1));
+    std::bind([](Telemetry::FlightMode flight_mode) {
+        std::cout << "FlightMode: " << Telemetry::flight_mode_str(flight_mode)
+                  << std::endl;
+
+    }, std::placeholders::_1));
 
     action_ret = device.action().takeoff();
     ASSERT_EQ(Action::Result::SUCCESS, action_ret);
 
     sleep_for(seconds(5));
 
-    ////////////////////////////////////////
-    /// Uses Default PX4 FollowMe config////
-    // Min height: 8mts
-    // Distance to device: 8mts
-    // Responsiveness: 0.5
-    // Device follows from: Behind
-    ////////////////////////////////////////
-    auto defult_cfg = device.followme().get_config();
-    print_config(defult_cfg);
+    auto defult_config = device.followme().get_config();
```

Comment:
default_config ?

Reply:
if no config is set, it is default PX4 follow me config.

Reply:
Sorry, I mean that this is miss-spelled.